### PR TITLE
Fixes #15120: Error after upgrade from 4.1 to 5.0 on RPM because slapd is not restarted: apiAuthorizationKind: attribute type undefined

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/15120

### CODE REVIEW

File: rudder-inventory-ldap/debian/postinst [hunk 1]
```diff
@@ -33,6 +33,23 @@ case "$1" in
...
     # Restart rudder-slapd to take schema changes into account
     echo -n "INFO: Restarting rudder-slapd..."
+    # during upgrade, service may not have been started with systemctl
+    # detect first if rudder-slapd is seen running ?
+    RUDDER_SLAPD_RUNNING_SERVICE=0
+    RUDDER_SLAPD_RUNNING_SYSTEMCTL=0
+    if service rudder-slapd status > /dev/null; then 
+      RUDDER_SLAPD_RUNNING_SERVICE=1
+    fi
+    if systemctl status rudder-slapd > /dev/null; then
```
Comment (amousset) [inline rudder-inventory-ldap/debian/postinst]: we should use is-active instead

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by amousset** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**